tensorflow/python/lib/io/file_io_integration_test.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import tensorflow as tf
```

Comment:
Is there a way to change this to not import TF fully? As it is written this raises this error (sanitized):

```
HOURGLASS IMPORT DETECTED

'import tensorflow' statements are no longer allowed in the internal
TensorFlow codebase, with the exception of /examples/ and /g3doc/
directories. This is because they prevent Bazel and <testing tool> from caching
anything.

Please import modules directly. Also please make sure your Python
BUILD rules depend on all the labels whose sources they import.
```

Reply:
@mihaimaruseac Let me take a look. I think it should be possible. The reason was to enable all modules (so that in case multiple copies of `Env::Default` is present in the future again, it will trigger the test fail). I will see if I can find an internal import that can achieve the same thing.

Reply:
Alternatively, maybe we can change [the nightly smoke test](https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/tools/ci_build/builds/nightly_release_smoke_test.sh;l=24;drc=82f83aae97ae11736e577fa560e9d33cf764a018) to test for this instead. It will prevent nightlies from being pushed and will still give a signal.

Reply:
Thanks @mihaimaruseac . Let me see if I can update the PR.